auth0_flutter_platform_interface/lib/src/auth0_flutter_web_platform.dart

```diff
@@ -1,7 +1,9 @@
 import 'package:plugin_platform_interface/plugin_platform_interface.dart';
 import '../auth0_flutter_platform_interface.dart';
 
-class StubAuth0FlutterWeb extends Auth0FlutterWebPlatform {}
+class StubAuth0FlutterWeb extends Auth0FlutterWebPlatform {
+  StubAuth0FlutterWeb() : super();
```

Comment:
I added this line, so that the super invocation of `Auth0FlutterWebPlatform()` can be tested.